calour/experiment.py

```diff
@@ -64,7 +65,7 @@ class Experiment:
         The metadata on the samples
     feature_metadata : pandas.DataFrame
         The metadata on the features
-    exp_metadata : dict
+    metadata : dict
```

Comment:
the name metadata is a bit confusing with metadata used in the filtering functions (for the feature/sample metadata).
Also a bit confusing with exp_metadata/feature_metadata. Maybe keep it "exp_metadata"? or alternatively another name? or change the name of the "metadata" var used in the filtering functions etc.?